a/amazon-metadata.yml

```diff
@@ -0,0 +1,19 @@
+amazon-metadata:
+  image: rancher/os-amazon-metadata:v0.9.2${SUFFIX}
```

Comment:
unless the image needs to be updated every RancherOS release, I'd use a different versioning scheme. I'm working (slowly :( ) to decouple image building from releases.